azure-pipelines.yml

```diff
@@ -115,6 +115,13 @@ jobs:
           artifactName: packages
           artifactType: container
 
+      - task: PublishBuildArtifacts@1
```

Comment:
If publishing the tests like this you can remove `<IsPackable>true</>` from `NotIntegrationTests.csproj`.